packages/tfchain_client/src/dao.ts

```diff
@@ -101,6 +108,21 @@ class QueryDao {
     }
     return { active: activeProposals, inactive: inactiveProposals };
   }
+  @checkConnection
+  async getVotesOfProposal(hash: string): Promise<ProposalVotes> {
```

Comment:
Can we change this to `getProposalVotes`?

Reply:
I just noticed we already have a function with the same name. I suggest editing the function to include the progress as well.

https://github.com/threefoldtech/tfgrid-sdk-ts/blob/15bdc6225a77589c9510d42f69cc362b8de2e4c1/packages/tfchain_client/src/dao.ts#L161-L166

Reply:
we can't use ```getProposalVotes``` directly as it's a private method so it used inside ```getVotesOfProposal``` .

Reply:
We can convert it to public instead of implementing a new method from scratch, no?